roles/datadog/handlers/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+
+- name: restart datadog
+  action: service name=datadog-agent state=restarted
```

Comment:
Does this need to be `sudo: true`. I got these errors trying to run this branch.

NOTIFIED: [datadog | restart datadog] ***************************************** 
failed: [ec2-52-24-229-252.us-west-2.compute.amazonaws.com] => {"failed": true}
msg: rm: cannot remove ‘/var/run/datadog-supervisord.pid’: Permission denied
rm: cannot remove ‘/var/lock/subsys/datadog-agent’: Permission denied
Password: su: Authentication failure
rm: cannot remove ‘/var/run/datadog-supervisord.pid’: Permission denied
rm: cannot remove ‘/var/lock/subsys/datadog-agent’: Permission denied